games/commander-genius/.gitignore

```diff
@@ -0,0 +1 @@
+/commander-genius*
```

Comment:
Ususally, I don't put stiff in gitignore, this lets me see easily what was added by build scripts and clean everything up with git clean -df